spec/unit/knife/core/windows_bootstrap_context_spec.rb

```diff
@@ -220,6 +220,8 @@
       let(:bootstrap) { Chef::Knife::Bootstrap.new(["--bootstrap-install-command", "chef-client"]) }
       before do
         bootstrap.config[:bootstrap_install_command] = "chef-client"
+        @old_env = ENV["CHEF_LICENSE"]
+        ENV["CHEF_LICENSE"] = "accept"
```

Comment:
I would prefer `accept-no-persist` if possible, so we don't accidently leave a marker file around and then have a test erroneously pass later

Reply:
I'll update this in the next PR I'm going to write which fixes `knife bootstrap` to only perform the license acceptance flow if the remote version is >= 15